src/App.js

```diff
@@ -1,14 +1,70 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {Navigate, Route, Routes} from "react-router-dom";
 import useAuth from "auth/useAuth";
 import {CssBaseline, ThemeProvider} from "@mui/material";
 import {defaultTheme} from "assets/styles";
-import Home from "pages/Home";
 import Login from "pages/Login";
 import SignUp from "pages/SignUp";
 import ResetPassword from "pages/ResetPassword";
 import Footer from "components/Footer";
 import Error from "pages/Error";
+import Home from "pages/Home/Home";
+import highlight_1 from '../src/assets/images/highlight_1.svg';
+import highlight_2 from '../src/assets/images/highlight_2.svg';
+import highlight_3 from '../src/assets/images/highlight_3.svg';
+import highlight_4 from '../src/assets/images/highlight_4.svg';
+import CustomizedSnackbars from "components/Snackbar";
+
+export const projectTeam = [
+    {
+        name: 'Katsiaryna',
+        imageURL: 'https://icons.veryicon.com/png/o/internet--web/prejudice/user-128.png',
+        role: 'frontend team',
+        gitHub: 'https://github.com/katsiarynalashcheuskaya'
+    },
+    {
+        name: 'Alina',
+        imageURL: 'https://icons.veryicon.com/png/o/internet--web/prejudice/user-128.png',
+        role: 'backend team',
+        gitHub: 'https://github.com/npnote8'
+    },
+    {
+        name: 'David',
+        imageURL: 'https://icons.veryicon.com/png/o/internet--web/prejudice/user-128.png',
+        role: 'frontend team',
+        gitHub: 'https://github.com/DavidGslade86'
+    },
+    {
+        name: 'Bino',
+        imageURL: 'https://icons.veryicon.com/png/o/internet--web/prejudice/user-128.png',
+        role: 'backend team',
```

Comment:
Repeated strings should generally be moved to constants, so here `const BACKEND_TEAM='backend team'` or something like that.

Reply:
Done &#9989;